app/routes/auth.py

```
@@ -1,30 +1,22 @@
-from app.security import get_client_ip
-from app.compliance import ensure_kyc_verified
+from app.models import User, Wallet
 
 router = APIRouter()
 pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")
 oauth2_scheme = OAuth2PasswordBearer(tokenUrl="/api/auth/login")
 
-settings = get_settings()
+SECRET_KEY = os.getenv("JWT_SECRET", "changethis")
+ALGORITHM = os.getenv("JWT_ALGORITHM", "HS256")
+ACCESS_TOKEN_EXPIRE_MINUTES = int(os.getenv("ACCESS_TOKEN_EXPIRE_MINUTES", 60))
```

Ruff (0.15.6)

[warning] Invalid type for environment variable default; expected `str` or `None`

(PLW1508)